definitions/ext-rf_scanner/definition.yml

```diff
@@ -0,0 +1,51 @@
+#  [mandatory] The domain of an entity, default to EXT unless otherwise indicated by the maintainers.
+domain: EXT
+# [mandatory] The type of an entity. Must be unique within its domain, but may be duplicated between domains.
+# Please add a brief description of what your type is here on the definition file.
+type: RF-Scanner
+
+# Entity definitions without this section are synthesized based on rules defined elsewhere.
+synthesis:
+  # [mandatory] The name of a telemetry attribute that will be used as the id of the entity, so it needs to be unique,
+  # it can be the same field used for the name or not.
+  identifier: macAddress
+
+  # [mandatory] The name of a telemetry attribute that will be used as the name of the entity (i.e. k8s.cluster.name).
+  name: RF-Scanner
```

Comment:
Like the identifier this should also be a telemetry attribute, `macAddress` would be a good option.